src/EnergyPlus/InternalHeatGains.cc

```diff
@@ -7506,6 +7557,30 @@ namespace InternalHeatGains {
         SumCO2GainRate = tmpSumCO2GainRate;
     }
 
+    // Added for hybrid model -- function for calculating CO2 gains except people
+    void SumAllInternalCO2GainsExceptPeople(int const ZoneNum, // zone index pointer for which zone to sum gains for
```

Comment:
I feel like some reviewers' gonna ask why these functions are highly duplicated and code cannot be reused. But we can leave this as a open discussion for now, unless you feel it's easy to modify the code.